Fix various issues found by translators (#4297)
WeblateOrg/weblate: docs/admin/machine.rst

Machine translation service provided by Google Cloud services.

> This service differs from the former one by having another mean of authentication.

Reviewer: ```suggestion
This service differs from the former one by having another means of authentication.
```
"[(functioning as singular or plural) the medium, method, or instrument used to obtain a result or achieve an enda means of communication](https://www.dictionary.com/browse/means)"

Can this second method use the same way of authentication as the first? Otherwise it is an awkward construction.

> This service differs from the former one by having another mean of authentication.

Reviewer: ```suggestion
This service differs from the former one in how it authenticates.
```
